tests/schemas/api_v3/regex.js

```diff
@@ -31,6 +31,8 @@ const CHAIN_ID = /^\b[a-fA-F0-9]{8}\b$/;
 const TOKEN_ID = /^\b[a-fA-F0-9]{16}\b$/;
 const DURATION = /^\d{4}-\d{2}(?:-\d{2})?$/;
 const DIGITS = /^\d+$/;
+const HEX = /^\b[0-9a-fA-F]+\b$/;
+const TOPIC = /^\b(?:[0-9a-fA-F]+|lsk[a-hjkm-z2-9]{38})\b$/;
```

Comment:
Could you please also add the same regex in gateway for both the `/events` and `/transactions/dryrun` endpoints?

Reply:
Added for events. topics are sent in response of dryrun.